### PR TITLE
Update / Correct zamg integration description

### PR DESCRIPTION
**Pull request by killer0071234**
## Proposed change

The current description for the zamg integration has wrong sensors in.
In addition the introduction is using the wrong api link.
## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/zamg.markdown [hunk 1]
```diff
@@ -18,7 +18,7 @@ ha_codeowners:
 
 The `zamg` platform uses meteorological details published by the Austrian weather service [Zentralanstalt für Meteorologie und Geodynamik (ZAMG)](https://www.zamg.ac.at).
 
-Only observations for capital cities are publicly available. You can check the list of stations in [CSV format](https://www.zamg.ac.at/ogd).
+With this integration it is possible to get weather station data from up to 228 different zamg stations across Austria. The integration is using the [datahub-api](https://dataset.api.hub.zamg.ac.at/v1/docs/index.html). The old `odg` api (before 2022.11.1) was only able to read from 22 mayor stations.
```
Comment (frenck) [inline source/_integrations/zamg.markdown]: We don't have to keep a changelog here. Let's remove "the old" part.

File: source/_integrations/zamg.markdown [hunk 2]
```diff
@@ -34,18 +34,20 @@ This integration provides the following sensors:
...
-|Station Elevation|The station elevation in m|
-|Update Date|Update date of last read data|
-|Update Time|Update time of last read data|
-|Station id|The station id|
+|Sun Last 10 Minutes|Sunshine in the last 10 minutes in sec|
+|Precipitation|Precipitation in mm|
+|Snow|Snow in cm|
+
+<div class='note'>
+Not every station supports every Sensor.
```
Comment (frenck) [inline source/_integrations/zamg.markdown]:
```suggestion
Not every station supports every sensor.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/zamg.markdown (hunk 1 above)
Comment (killer0071234) [reply source/_integrations/zamg.markdown]: ok, done

File: source/_integrations/zamg.markdown [hunk 3]
```diff
@@ -18,7 +18,7 @@ ha_codeowners:
 
 The `zamg` platform uses meteorological details published by the Austrian weather service [Zentralanstalt für Meteorologie und Geodynamik (ZAMG)](https://www.zamg.ac.at).
 
-Only observations for capital cities are publicly available. You can check the list of stations in [CSV format](https://www.zamg.ac.at/ogd).
+With this integration it is possible to get weather station data from up to 228 different zamg stations across Austria. The integration is using the [datahub-api](https://dataset.api.hub.zamg.ac.at/v1/docs/index.html).
```
Comment (frenck) [inline source/_integrations/zamg.markdown]:
```suggestion
With this integration, it is possible to get weather station data from up to 228 different ZAMG stations across Austria. The integration uses the [datahub-api](https://dataset.api.hub.zamg.ac.at/v1/docs/index.html).
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @killer0071234 👍

**Pull request merged**